docs/reporters/AntennaAttributionDocumentReporter.md

```diff
@@ -0,0 +1,44 @@
+# AntennaAttributionDocumentReporter
+The AntennaAttributionDocumentReporter generates an attribution document in .pdf format by leveraging the [Eclipse 
+Antenna][1] project. It generates a blank PDF file without any configuration, but the attribution document can be 
+customized by providing a template JAR, or a directory with template PDF files. 
+
+There is a guide in the [Antenna project documentation][2], which describes how such a template JAR file for the 
+customization has to be created. This has to be done before using the reporter. 
+
+In addition to the JAR file technique, there is an alternative way to customize the attribution document. For this, a 
```

Comment:
"I addition to using a template bundle JAR, ..."

Reply:
Done.